open-metadata-implementation/access-services/data-engine/data-engine-server/src/main/java/org/odpi/openmetadata/accessservices/dataengine/server/handlers/DataEngineCollectionHandler.java

```diff
@@ -39,18 +39,18 @@
      * @param repositoryHelper              provides utilities for manipulating the repository services objects
      * @param dataEngineRegistrationHandler provides calls for retrieving external data engine guid
      * @param dataEngineCommonHandler       provides utilities for manipulating entities
-     * @param transformationProjectHandler  handler for managing schema attributes in the metadata repositories
+     * @param collectionOpenMetadataAPIGenericHandler  handler for managing schema attributes in the metadata repositories
```

Comment:
Description is not adequate. Instead proposing: collection handler helps building model for creating Collection metadata associated with Process assets.